instagram/insta/src/components/CommentForm.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'; 
+// import CommentSection from './CommentSection/CommentSection';
+
+class CommentForm extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            newComments: ""
+        }
+    }
+
+    handleChange = event => {
+        this.setState({ [event.target.name]: event.target.value })
+    }
+
+    render() {
+        return (
+            <div>
+                <form onSubmit={this.props.addNewComment(this.props.postId, this.state.newComment)}>
+                    <input  
+                        type='text'
+                        placeholder ='Add a comment'
+                        value={this.state.newComment}
+                        onChange={this.handleChange}
```

Comment:
your value doesnt match your state value, on state you have it plural, newComments